controller/engine_image_controller.go

```diff
@@ -42,6 +42,8 @@ var (
 type EngineImageController struct {
 	*baseController
 
+	logger logrus.FieldLogger
+
```

Comment:
you don't need the `logger` field, it's already covered by the embeeded *baseController.
The PR longhorn/longhorn#662 is a little older already and it came with the baseController refactoring.